src/index.js

```diff
@@ -17,34 +20,28 @@ const Container = styled.div`
   min-width: 100vw;
 `;
 
-const rootElement = document.getElementById("root");
+const App = () => {
+  const [async, setAsync] = useState();
```

Comment:
Am I a bastard if I ask to rely on `isAsync` and `useState(false)` ? 🙄

Reply:
My bad, I was thinking of initializing elsewhere at first then changed my mind...